src/v2/view-builder/internals/BaseForm.js

```diff
@@ -45,9 +45,8 @@ export default Form.extend({
   },
 
   handleClearFormError() {
-    if (this.$('.o-form-error-container').hasClass('o-form-has-errors')) {
-      this.clearErrors();
-    }
+    this.clearErrors();
+    this.introspectMessage && this.introspectMessage.remove();
```

Comment:
baseForm should not be aware of view specific actions..
can you try here instead 
```
    this.$el.find('.okta-form-infobox-error').remove();
```

Reply:
Yes, but the error callout has a div wrap around the `.okta-form-infobox-error`. So removing `.okta-form-infobox-error` only does not clean up all the doms.

Reply:
Or I can rename `introspectMessage` to `calloutMessage` so it is less specific to the flow. What do you think?

Reply:
I was wondering this should have removed any existing message when user hit `save`.. 
https://github.com/okta/okta-signin-widget/pull/2196/files#diff-75984ad57a18502463b8e54df919333e998b41678372612bc528336229069c5bR58


Isn't the issue was that overridden `showMessages` function.. what happen if you remove that function ?

Reply:
Hitting `save` removes the existing errors. 

Removing `showMessages` from `IdentifierView` does not work because the fall back method from `BaseForm` shows the message in plain text, instead of in a callout format.

Reply:
Cool. so i feel the right solution to just override the showMessages in IdentifierView with a function that renders error callout.. We can safely remove any `warning` types because there is no `warn` in backend..